### PR TITLE
[RDY] Rename frontend classes

### PR DESCRIPTION
**Pull request by Unia**
~~Depends on #189.~~

### CODE REVIEW

**Comment by pvanagtmaal** [comment]
Renaming looks fine to me. I think all classnames make sense now.

**Comment by Unia** [comment]
I will rebase this as soon as #189 is merged.

File: dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/controllers/WelcomeController.java [hunk 1]
```diff
@@ -29,7 +30,7 @@
 	private ObservableList<String> databases;
 
 	private DBLoadService dbload;
-	private DirectoryLoadService dirload;
+	private DirectoryListingService dirload;
```
Comment (Balletie) [inline dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/controllers/WelcomeController.java]: I agree.

**Comment by Balletie** [comment]
Yes, these names make more sense. Merging.

**Pull request merged**